uaclient/tests/test_util.py

```diff
@@ -350,6 +350,43 @@ def test_retry_logs_remaining_retries(self, m_sleep, m_subp, caplog_text):
         for log in expected_logs:
             assert log in logs
 
+    @pytest.mark.parametrize("caplog_text", [logging.WARNING], indirect=True)
+    @mock.patch("uaclient.util._subp")
+    def test_log_subp_fails_stdout_stderr_capture_true(
```

Comment:
Maybe we can combine both of these tests and just parametrize the `capture` parameter

Reply:
Yes, I was reading more on pytest just about that.

Reply:
Updated.